cmd/manager/main.go

```diff
@@ -161,9 +164,40 @@ func main() {
 			},
 		}
 	}
+
+	// http/2 should be disabled due to its vulnerabilities. More specifically,
+	// disabling http/2 will prevent it from being vulnerable to the HTTP/2 Stream
+	// Cancellation and Rapid Reset CVEs. For more information see:
+	// - https://github.com/advisories/GHSA-qppj-fm5r-hxr3
+	// - https://github.com/advisories/GHSA-4374-p667-p6c8
+	disableHTTP2 := func(c *tls.Config) {
+		setupLog.Info("disabling http/2")
+		c.NextProtos = []string{"http/1.1"}
+	}
+
+	tlsOpts = append(tlsOpts, disableHTTP2)
```

Comment:
It appears that TLSv1.2 is the default minimum for TLS version, so we probably don't need to add an explicit minimum.

Reply:
That is an amazing good point for us to validate!

Reply:
I think you are right. But, we need to set the above code  "http/1.1 to prevent the CVE

Reply:
Those CVEs are for golang.org/x/net below version 0.17.0 and grpc below version 1.58.3. We are on 0.31.0 net and 1.67.1 grpc so we don't need to do anything here.

Reply:
OH thank you for check that so that is something that we can remove great !!!!